README.md

```diff
@@ -61,11 +61,11 @@ Three build systems are supported:
 
 Compiling with [conan](https://conan.io/):
 
-Assuming that you are in the **parent** directory of `BehaviorTree.CPP`:
+Assuming that you are in the **root** directory of `BehaviorTree.CPP`:
 
 ```
 mkdir build_release
-conan install . -of build_release -s build_type=Release
+conan install . -of build_release -s build_type=Release --build=missing
```

Comment:
`--build=missing` intructs conan to build whatever package is missing. For example I worked on this PR on Ubuntu 24.04 using gcc13 and conan-center had no binaries to serve me. The usage of this flag is trivial knoledge for someone who knows this way around conan, but it can be very helpul for someone who is taking his first steps with it.